docs/tasks.md

```
@@ -769,39 +785,48 @@ WebSocketを使用したリアルタイム更新を統合する
+   ```json
+   {
+     "bin": {
+       "claude-work": "./dist/bin/cli.js"
+     }
+   }
+   ```
+2. `src/bin/cli.ts`作成（カスタムサーバー起動）
+3. `tsconfig.json`のoutDirを`dist/`に設定
+4. ビルドスクリプト作成: `"build": "tsc && cp -r public dist/"`
+5. `.env.example`作成
+6. `npm run build && npx . `でローカルテスト
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)